### PR TITLE
Dynamically routed users via auth-0

### PR DESCRIPTION
**Pull request by quinnha**
https://github.com/smiths/capTemplate/assets/35006641/df2128b2-d6ad-4df2-8fa3-f8d6a466a2db

Dynamically routed users via auth-0, showing different satellites per users as per mongoDB record.

#214

### CODE REVIEW

**Comment by quinnha** [comment]
Need to auto add users to db when they get registered, need to be fixed in another pr when auth0 decides to work

File: src/leo-client-app/src/components/SatellitesOfInterest.tsx [hunk 1]
```diff
@@ -35,7 +34,10 @@ type SatelliteDetails = {
 };
 
 const SatellitesOfInterest = ({ userId }: Props) => {
-  const satellites = useGetUserSatellites(userId);
+  const satellites = useGetUserSatellites(
+    userId ? userId : "66059b2aa430445956d73120"
```
Comment (Umang-Rajkarnikar) [inline src/leo-client-app/src/components/SatellitesOfInterest.tsx:38]: Do we need this conditional check?

**Review by Umang-Rajkarnikar — APPROVED** [review]
lgtm, just had a comment

File: src/leo-client-app/src/components/SatellitesOfInterest.tsx (hunk 1 above)
Comment (Umang-Rajkarnikar) [reply src/leo-client-app/src/components/SatellitesOfInterest.tsx:38]: ah gotcha

**Pull request merged**